ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/EntityUtil.java

```diff
@@ -15,13 +15,30 @@
 
 package io.confluent.ksql.rest.util;
 
+import avro.shaded.com.google.common.collect.ImmutableMap;
 import io.confluent.ksql.rest.entity.FieldInfo;
 import io.confluent.ksql.rest.entity.SchemaInfo;
 import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 import org.apache.kafka.connect.data.Schema;
 
 public final class EntityUtil {
+  private static final Map<Schema.Type, Function<Schema, SchemaInfo.Type>>
+      SCHEMA_TYPE_TO_SCHEMA_INFO_TYPE =
+      ImmutableMap.<Schema.Type, Function<Schema, SchemaInfo.Type>>builder()
```

Comment:
nit: why does this need to be a function (`s` is never used)?

Reply:
Ah true. However, it's gonna be used in a follow-up patch to support decimals. The Schema that will be used is BYTES, and the function will check the logical type name matches a Decimal type. This logical type name is obtained from s.name().

Even if it is not used in this patch, can I leave this code so I don't do more changes for decimals?

Reply:
I actually removed the unused function<> for this patch just to keep it clean.